core-dev/packages/coq/coq.dev/files/coq.install

```diff
@@ -1,12 +0,0 @@
-bin: [
-  "bin/coqwc"
```

Comment:
I'd suggest updating this to _build/install/default/bin/coqwc etc...

Reply:
coqide.install also needs update.

Reply:
@ejgallego isn't it easier if you just push to the branch for this PR (or make your own PR)?^^

Reply:
Sure @ejgallego, as I said before, feel free to take over this PR. For my part, since I don't see the point of install files when you already have an install target, I've just removed the install file for CoqIDE as well. If it works, I'll consider that my job is done.